components/runtime/squeak4/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Package Makefile for the "Classic VM" Squeak Smalltalk V4 system
+# See http://squeak.org and http://squeakvm.org
+#
+# The squeak4 vm can only run Squeak4 images; it will not work with 
+# Squeak5 images; squeak5 uses  different technology called "Cog Spur"
+#
+# squeak4 uses cmake and a custom configure script (not GNU autoconf)
+# in order to build this package, "cmake" must be installed
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		squeak4
+COMPONENT_VERSION=	4.16.7
+COMPONENT_SUMMARY=	The Squeak V4 Smalltalk Virtual Machine
+COMPONENT_PROJECT_URL=	http://www.squeak.org
+COMPONENT_ARCHIVE_URL=  http://squeakvm.org
+COMPONENT_FMRI=		runtime/squeak4
+COMPONENT_CLASSIFICATION=	Development/Other Languages
+SVN_REPO=		http://squeakvm.org/svn/squeak/trunk/
+SVN_REV=		3775
+SVN_HASH=  sha256:1f40ae6a9ad05739c07d63f4e67bfd6afacbe76c4deb2802bba1be7332f373a7
+
+# See http://wiki.squeak.org/squeak/933
+# See http://wiki.squeak.org/squeak/159
+COMPONENT_LICENSE=      MIT
+COMPONENT_LICENSE_FILE= squeak4.license
+
+
+TEST_TARGET= $(NO_TESTS)
+
+include $(WS_MAKE_RULES)/common.mk
+
+PATH=$(PATH.gnu)
+
+# squeak4 configure looks for .svn or svnversion file, but
+# prep-svn.mk did a svn export and created an unversioned copy of the sources
+COMPONENT_PRE_CONFIGURE_ACTION = ( echo $(SVN_REV) > $(SOURCE_DIR)/platforms/unix/svnversion)
+
+CONFIGURE_SCRIPT= $(SOURCE_DIR)/platforms/unix/cmake/configure
+CONFIGURE_OPTIONS +=	--src=$(SOURCE_DIR)/src
+
+REQUIRED_PACKAGES += service/opengl/ogl-select
```

Comment:
Why is ogl-select in this list? Should it be mesa?

Reply:
To be checked ...  I believe that "gmake REQUIRED_PACKAGES" did not find or discover mesa.

Reply:
Changed to mesa.